manifests/cf-manifest/manifest/020-cf-properties.yml

```diff
@@ -506,11 +506,14 @@ properties:
       auctioneer:
         api_url: http://auctioneer.service.cf.internal:9016
       etcd:
-        machines: [(( grab properties.etcd.advertise_urls_dns_suffix ))]
-        ca_cert: (( grab meta.secrets.consul_ca_cert ))
-        client_cert: (( grab secrets.consul_agent_cert ))
-        client_key: (( grab secrets.consul_agent_key ))
+        machines: []
+        ca_cert: ""
```

Comment:
you could change it then to st like "not_used" - to make it obvious that this has to be here, but is not being used

otherwise that looks like many of those null variables we have around manifest...